src/ConfigManager.cpp

```diff
@@ -235,13 +268,34 @@ void ConfigManager::setup() {
     startAP();
 }
 
+void ConfigManager::reset() {
+    // char magic[2];
+    char ssid[32];
+    char password[64];
+
+    for (int i = 0; i < sizeof(ssid); i++) {
+        ssid[i] = 0;
+    }
+    
+    for (int i = 0; i < sizeof(password); i++) {
+        password[i] = 0;
+    }
+
+    EEPROM.put(0, 0);
+    EEPROM.put(WIFI_OFFSET, ssid);
+    EEPROM.put(WIFI_OFFSET + 32, password);
+    EEPROM.commit();
+
```

Comment:
Not sure if this PR is going anywhere, but I'd suggest breaking this call out into its own method (my example may not be complete) ... and refactor the couple other places the wifi creds get stored, as it is now in at least 2 places 

```
void UpdateWifiSettings(String ssid, String password) {
    strncpy(ssidChar, ssid.c_str(), sizeof(ssidChar));
    strncpy(passwordChar, password.c_str(), sizeof(passwordChar));

    EEPROM.put(0, magicBytes);
    EEPROM.put(WIFI_OFFSET, ssidChar);
    EEPROM.put(WIFI_OFFSET + 32, passwordChar);
    EEPROM.commit();
}
```

Reply:
Nice idea, what about calling it storeWifiSettings, or something like that. It would also need to take the actual char arrays (including the magic bytes array), so we overwrite the entire thing properly.